plugin/pkg/admission/podpreset/admission.go

```diff
@@ -235,6 +247,38 @@ func mergeEnvFrom(pip *settings.PodPreset, original []api.EnvFromSource) ([]api.
 	return append(original, pip.Spec.EnvFrom...), nil
 }
 
+func mergeAffinity(pip *settings.PodPreset, original *api.Affinity) (*api.Affinity, error) {
+
+	if original == nil {
+		return pip.Spec.Affinity, nil
+	}
+
+	if pip.Spec.Affinity != nil {
+
+		if pip.Spec.Affinity.NodeAffinity != nil {
```

Comment:
So the logic here means that I cannot merge pod Affinity if the pod already have Affinity, +1 to this logic, can you please add some comments for this why do not support update affinity with PodPreset?